File: week12/SacramentPlanner/SacramentPlanner/Models/MemberModel.cs
using System;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace SacramentPlanner.Models
{
    public class MemberModel
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MemberTitle { get; set; }

        public MemberModel(string firstName, string lastName, string memberTitle)
        {
            FirstName = firstName;
            LastName = lastName;
            MemberTitle = memberTitle;
        }

        public MemberModel() {}
    }
}

File: week12/SacramentPlanner/SacramentPlanner/ClientApp/src/app/members/member.model.ts
export class Member {
  public id: string
  public firstName: string
  public lastName: string
  public memberTitle: string

  constructor(firstName: string = "", lastName: string = "", memberTitle: string = "") {
    this.firstName = firstName
    this.lastName = lastName
    this.memberTitle = memberTitle
  }
}
